tardis/plasma/base.py

```diff
@@ -281,36 +297,81 @@ def write_to_dot(self, fname, latex_label=True):
             return
         print_graph = self.graph.copy()
         print_graph = self.remove_hidden_properties(print_graph)
+        edge_names = [e for e in print_graph.edges]
```

Comment:
why - regenerate a list?

Reply:
This was mostly easier for myself to read, but this is a fair point that I understand can be omitted.